string EngagementMode
string TrackingMode
---
move_base_msgs/MoveBaseGoal human
